docs/getting_started/gce.rst

```diff
@@ -5,7 +5,7 @@ Google Compute Engine
 
 As of Mantl 0.3 you can bring up Google Compute Engine environments using
 Terraform. Mantl uses Terraform to provision hosts. You
-can `download Terraform from terraform.io
+can 'download Terraform from 'terraform.io
```

Comment:
Both these quotes should be removed, and there should be a backtick \` before the word "download", just as it was before any changes were made.